src/lang/locales/ja-JP.json

```diff
@@ -88,6 +95,7 @@
     "close": "閉じる"
   },
   "sections": {
+    "attributes": "Attributes",
```

Comment:
Should we just remove this if we don't have a translation yet? The behavior is probably the same, just checking.

Reply:
It seems that this one didn't got into the translation process (it may be newer). So I deleted this key.